includes/admin/donors/donor-actions.php

```diff
@@ -124,6 +128,10 @@ function give_edit_donor( $args ) {
 	// Save company name in when admin update donor company name from dashboard.
 	$donor->update_meta( '_give_donor_company', sanitize_text_field( $args['give_donor_company'] ) );
 
+    $donorModel = Donor::find($donor->id);
+    $donorModel->phone = sanitize_text_field($args['give_donor_phone_number--international-format']);
+    $donorModel->save();
+
```

Comment:
@glaubersilva the one part i'm debating here is if using the model should be used directly in the legacy code as it's only used for that one update and could have a side effect.  

Another option is we could create a new action here that we can hook into and run the update with the model.  This would also give us more flexibility for adding more fields in the future.  Something like 
```
do_action('give_admin_donor_details_updating', $args, $donor->id);
````
```
add_action('give_admin_donor_details_updating', function($args, $donorId){
    $donorModel = Donor::find($donor->id);
    $donorModel->phone = sanitize_text_field($args['give_donor_phone_number--international-format']);
    $donorModel->save();
}, 10, 2)
```

Reply:
@jonwaldstein Thanks for the suggestion! I liked this idea because using the Donor model in the legacy code also bothered me.  

Resolved in 6fc5a89eab240dc2f4cfab6b6741171ed0fc7886 and c36a9fb73ea86e76d2c9a86b6916d04631cabfc2

Reply:
Great minds think alike! ✊